src/neuroconv/datainterfaces/behavior/deeplabcut/requirements.txt

```diff
@@ -1,2 +1,2 @@
-dlc2nwb>=0.2
+dlc2nwb==0.2
```

Comment:
Latest release of dlc2nwb causes some secondary dependence issues in the CI

Nothing to worry about, but they're fixed in #234 

Pinning here so CI can pass